include/config/general.h

```diff
@@ -80,4 +80,7 @@
 #define UNITS                        UNITS_IMPERIAL
 #define CHAR_DEC_SEPARATOR           CHAR_PERIOD // CHAR_PERIOD is used as a decimal separator only in the UK and the US. The rest of the world uses CHAR_COMMA.
 
+// Naming Screen
+#define AUTO_LOWERCASE_KEYBOARD      GEN_LATEST  // Starting in GEN_6, after entering the first uppercase character, the keyboard switches to lowercase letters;
```

Comment:
More of a nitpick if anything, but there's a semi colon at the end of the comment, instead of a comma.

Reply:
https://github.com/rh-hideout/pokeemerald-expansion/pull/4976/commits/888b0e0b44395e33a5b3ed9a00b96701ea4b1416